src/main/thrift/parquet.thrift

```diff
@@ -637,53 +643,32 @@ struct OffsetIndex {
  * Each <array-field>[i] refers to the page at OffsetIndex.page_locations[i]
  */
 struct ColumnIndex {
-/**
- * A list of bools to determine the validity of the corresponding min and max
- * values. If true, the page contains only null values so the corresponding
- * entries in min_values and max_values should be ignored.
- */
+  /**
+   * A list of bools to determine the validity of the corresponding min and max
+   * values. If true, the page contains only null values and the corresponding
+   * entries in min_values and max_values should be ignored. If false, the
+   * corresponding entries must be valid.
+   */
   1: required list<bool> null_pages
```

Comment:
We should specify that the min and max values for a page of only null values cannot be omitted from the min_values and max_values lists. Instead it should be byte[0] (because thrift doesn't allow null in the lists). This preserves the property that for page `i` in a column, its min is `min_values[i]` max is `max_values[i]`, etc.

Reply:
Done.